src/music_kraken/utils/shared.py

```diff
@@ -2,8 +2,11 @@
 import logging
 import tempfile
 import os
+import io
+import configparser
+from sys import platform as current_os
 
-
+USER_XDG_DIR_FILE = os.path.expanduser("~/.config/user-dirs.dirs")
```

Comment:
There is a system level fallback version of `user-dirs.dirs`, located at `/etc/xdg/user-dirs.defaults`. You should also read that one if the user one is not found. Note that it does not have to exist either.

Note that it is unclear to me what to do in the case both files exist, but only the system config file contains the value you want. I'd vote to use the system value if the user value does not override it, but you could also ignore the entire system config if the user config file exist.

Reference: https://www.freedesktop.org/wiki/Software/xdg-user-dirs/

Reply:
I'd say user-dirs.dirs is probably populated by most popular DE when setting up a new user with a language different than English.

On Arch,  `/etc/xdg/user-dirs.defaults` is a package required by the "plasma-desktop" meta package.